content/pages/features.html

```diff
@@ -5,152 +5,157 @@
         <meta name="url" content="features/">
     </head>
     <body>
-        <!-- dashboard -->
-        <section class="col-12 secondary">
-            <div class="col-12 screenshot-title">
-                <h3>A central dashboard to start your day</h3>
-            </div>
+        <div class="features">
```

Comment:
You can break the indenting rules a little bit here and indent with 2 spaces from the body tag and not indent the rest of the blocks. Let me see if I can review the rest easily as-is, if not you can apply the change I'm talking about.

Reply:
Hmmm, I have a better idea, why not apply the features class to the body tag instead ? If you're using it as a selector only then this will do.